### PR TITLE
[mlrun-kit] Add minio support

### PR DESCRIPTION
**Pull request by eliyahu77**
### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove irrelevant fields.]
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### Description:
add minio  support

### CODE REVIEW

File: stable/mlrun-kit/values-minio.yaml [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    requests:
+      memory: 0.5Gi
+  service:
+    type: NodePort
+    port: 9000
+    nodePort: 30080
+  consoleService:
+    type: NodePort
+    port: 9001
+    nodePort: 30090
```
Comment (Tankilevitch) [inline stable/mlrun-kit/values-minio.yaml:194]: It seems like minio also supports ingress - https://github.com/minio/charts/blob/main/minio/values.yaml#L178 
Can you please add it following how we are doing it for other charts https://github.com/v3io/helm-charts/blob/4bbea7cf5d2ca8b09d50ac00472cfb3212dd9222/stable/mlrun/values.yaml#L21

File: stable/mlrun-kit/values-minio.yaml [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  replicaCount: 0
+  storageClass:
+    create: false
+  rbac:
+    create: false
+  nameOverride: nfs-server-provisioner
+  persistence:
+    enabled: false
+    storageClass: "mlrun-kit-nfs"
+    size: 5Gi
```
Comment (Tankilevitch) [inline stable/mlrun-kit/values-minio.yaml:176]: Please add the following comment in line 161: 
`## nfs is not supposed to be activated, minio is used instead. For using nfs use values.yaml`

File: stable/mlrun-kit/README.md [hunk 3]
```diff
@@ -1,143 +1,161 @@
...
+## Chart Details
+
+The Open source MLRun kit chart includes the following stack:
+
+* Nuclio - https://github.com/nuclio/nuclio
+* MLRun - https://github.com/mlrun/mlrun
+* Jupyter - https://github.com/jupyter/notebook (+MLRun integrated)
+* NFS - https://github.com/kubernetes-retired/external-storage/tree/master/nfs
+* MPI Operator - https://github.com/kubeflow/mpi-operator
+* Minio - https://github.com/minio/minio/tree/master/helm/minio
```
Comment (Tankilevitch) [inline stable/mlrun-kit/README.md:14]: I think we should seperate NFS and Minio from this list and add it under **Supported storage solutions** title, feel free to offer different title name I am not quite sure about the naming.

File: stable/mlrun-kit/README.md [hunk 4]
```diff
@@ -1,143 +1,161 @@
...
+## Installing MLRun-kit on minikube
+The Open source MLRun kit uses node ports for simplicity. If your kubernetes cluster is running inside a VM, 
+as is the case when using minikube, the kubernetes services exposed over node ports would not be available on 
+your local interface, but instead, on the virtual machine's interface.
+To accommodate for this, use the `global.externalHostAddress` value on the chart. For example, if you're using 
+the kit inside a minikube cluster, pass the VM address in the chart installation command like so:
+
+```bash
+$ helm --namespace mlrun \
+    install my-mlrun \
```
Comment (Tankilevitch) [inline stable/mlrun-kit/README.md:61]: Add `## Installing MLRun-kit with NFS `as you did below for minio

**Review by Tankilevitch — CHANGES_REQUESTED** [review]
Looks good, mainly commenting comments.

**Review by Hedingber — CHANGES_REQUESTED** [review]
Although you didn't change all lines in each file, github shows as if you did, I guess it's related to end of line characters or something (because you're working with windows and we're with Mac), please try to fix it, we can't review like that, we can't see what actually changed

**Pull request closed without merging**